### ISSUE TITLE
suggestion: make -log default

### ISSUE THREAD

**Issue by TheBestPessimist** [gilbertchen/duplicacy#344]
From my understanding -log has the sole purpose of printing the timestamp.
I think that there should be no -log option, and the timestamp should be printed by default (as in "make -log always on, and hide the parameter). This would possibly help other people who are testing duplicacy atm.

**Comment by jonreeves** [comment]
Didn't realize there was a `-log` switch... Had to go poking about until I noticed it comes before the `command`. It appears to print a little more than Timstamp... looks like `Timestamp` `Log Level` `Reporter`

For my usage I'd probably keep it on all the time, but I can imagine that it could be a little verbose for some.

**Comment by TheBestPessimist** [comment]
I always keep it on in my scripts.

**Comment by TheBestPessimist** [comment]
It has some more info, true. Maybe I should rephrase my request:
At least the time should always be shown by default. As for log level and reporter, those should be shown with the -d and -v options.

**Comment by gilbertchen** [comment]
I'm not sure which one is more right, but at least this is a doc issue.  There should be a wiki page on the so called Global Options.  I promise it should be done by the end of tomorrow if not today.

**Comment by TheBestPessimist** [comment]
I see it has been done :).

Anyhow: i have looked at the logs which duplicacy makes for my setup (i always  use `-log`), and i think that the timestamp is the only part which i _**must have**_ always, the other 2 parts are not really needed. 

I have also seen a few other scripts, and they all have the `-log` option enabled (no `-v` or `-d`).

As this is the case, i'm thinking that the timestamp should be shown by default. It just feels better this way.

But if that will be implemented then a new question comes: what should happen with -log?
From my testing/usage, when i used `-d` (honestly, i never used `-v` <-- it feels like a "not needed" parameter), i always needed the `-log` and i was looking at both the "source of that log" (eg. `WARN SKIP_FILE ....`) and the specific message. 
Hence this is why I am proposing that the remaining of `-log` should be added both to `-v` and `-d` and the parameter should be removed.

Even more: when i think of an option named `log` i was actually expecting that this would write to a file, not parse the output in some way.

**Comment by TheBestPessimist** [comment]
Related: https://forum.duplicacy.com/t/a-bit-more-log-info/482/

**Comment by archon810** [comment]
I agree that the timestamps should be on by default for all output, even without `-log`.
